### ISSUE TITLE
Make Block content accessible via screen reader

### ISSUE THREAD

**Issue by ragesoss** [WikiEducationFoundation/WikiEduDashboard#1093]
Currently, the Trix editor has an accessibility bug that makes it unusable with JAWS screen reader. This means Wiki Ed's classroom program manager cannot make changes to course timeline content.

Until Trix itself improves (#606), we should provide an accessible fallback option for editing block text.

**Comment by ragesoss** [comment]
This is hopefully fixed with the switch to TinyMCE in place of Trix.
